### PR TITLE
Wpf to Angular edits

### PR DESCRIPTION
**Pull request by brianlagunas**
Made some small changes to the WPF to Angular guidance topics.
- embedded the related videos at the top of each topic.
- made small text edits to improve readability and flow.

### CODE REVIEW

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ _keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular,
 
 ## Responding to Events
 
+> [!Video https://www.youtube.com/embed/V1Futz4W400]
```
[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: You wouldn't happen to know the syntax for that would you? the DocFx documentation is lacking in this area.

[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: also, do you have a recommended width?

[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: I looked at the docfx code base and it does not appear that you can set the width of the video. Keep in mind that the content under the videos are a transcript of the actual video. It's okay if the videos are the dominate content in the post.

File: en/components/general/wpf_to_angular_guide/create_ui_with_components.md [hunk 2]
```diff
@@ -36,15 +38,15 @@ Now that we've seen just how similar an Angular component is to a WPF user contr
 ng g c sample
 ```
 
-When the command completes, you'll notice a number of things have happened. First, we have a new folder with the name we gave to our component. We also have four new files - an html file, a SPEC file, a typescript file, and a CSS file. We also made an update to the `app.module.ts` file.
+When the command completes, you'll notice a number of things have happened. First, we have a new folder with the name we given to our component. We also have four new files - an html file, a SPEC file, a typescript file, and a CSS file. We also made an update to the `app.module.ts` file.
```
[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/create_ui_with_components.md]: fixed

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 3]
```diff
@@ -68,7 +70,7 @@ onInputKeyup(event, message) {
 }
 ```
 
-However, in Angular there is an easier way to do that, you could bind to the `keyup.enter` pseudo-event. Using this approach, the event handler will be called only when the user presses Enter.
+Surprisngly, in Angular, there is an even easier way to do that. You could bind to the `keyup.enter` pseudo-event. Using this approach, the event handler will be called only when the user presses Enter.
```
[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: fixed

File: en/components/general/wpf_to_angular_guide/angular_events.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tachojelev) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: The **md** files get translated to html, so we can put a simple iframe (as we do for generating the live samples). The [!Video] tag is translated to iframe as well and youtube provides a way to copy an embedded link with iframe from the **share** button on their website.
In regards to the width, @hanastasov might have some ideas.
A working link as an example could be:
```html
<div style="height:350px">
    <iframe width="560" height="315" src="https://www.youtube.com/embed/V1Futz4W400" frameborder="0" allowfullscreen></iframe>
</div>
```

[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: We shouldn't let the size of the video stop us from merging this PR. We can work on a better sizing option later. These videos are the heart of our WPF to Angular guidance.

[the remark this answers is not in the document]
Comment (brianlagunas) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: By using the F12 tools, I did find 2 styles we might be able to use to control the size on a page by page basis:
```
div.embeddedvideo {
    width: 480px;
    height: 270px;
}

div.embeddedvideo iframe {
    width: 480px;
    height: 270px;
}
```

**Pull request merged**